utils/ffmpeg_install.py

```diff
@@ -77,10 +75,13 @@ def ffmpeg_install():
         # Try to run the FFmpeg command
         subprocess.run(['ffmpeg', '-version'], check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     except FileNotFoundError as e:
-        if not os.path.exists("./results") :
+        # Check if there's ffmpeg.exe in the current directory
+        if os.path.exists("./ffmpeg.exe"):
```

Comment:
not making any sense

Reply:
Yes it is, to check if it already has been downloaded